infra/bridge/test/email.js

```diff
@@ -51,34 +55,17 @@ describe('email attestations', () => {
   })
 
   it('should generate attestation on valid verification code', async () => {
-    const now = new Date()
-
-    // Fake a session
-    const parentApp = express()
-    parentApp.use((req, res, next) => {
-      req.session = {}
-      next()
-    })
-    parentApp.post('/api/attestations/email/verify', (req, res, next) => {
-      req.session.emailAttestation = {
-        // origin@protocol.foo
-        emailHash:
-          '$2b$10$aTtwdUXxMmlC1NQYwr7rI.afspPUJ32M/42PW8DNZmm8DMP5b6MQy',
-        code: 123456,
-        expiry: now.setMinutes(now.getMinutes() + 10)
-      }
-      next()
-    })
-    parentApp.use(app)
+    await client.set('origin@protocol.foo', 123456)
 
-    const response = await request(parentApp)
+    const response = await request(app)
       .post('/api/attestations/email/verify')
       .send({
         email: 'origin@protocol.foo',
         code: '123456',
         identity: ethAddress
       })
-      .expect(200)
+
+    // .expect(200)
```

Comment:
Why commenting out the check on response code ?

Reply:
Debugging 😬 fixed